Utils/Campaign builder/Unit1.pas

```diff
@@ -251,6 +255,12 @@ procedure TForm1.btnLoadCMPClick(Sender: TObject);
   fSelectedMap := -1;
   fSelectedNode := -1;
 
+  //From UpdateList ruled here since caused a bug with entering
+  //More details about the bug:
+  {http://gifok.net/images/2017/02/14/2017-02-14_16-01-27.gif}

```

Comment:
Please don't write descriptions of your changes and links to gifs in the code. Post them in the description of the pull request.

In this case I would say this change doesn't require a comment in the code (instead post the gif and reasoning for your changes in the pull request description).